packages/dcos-ui/buildinfo.json

```diff
@@ -3,7 +3,7 @@
   "single_source" : {
     "kind": "git",
     "git": "https://github.com/dcos/dcos-ui.git",
-    "ref": "89b9fbd36582d107e67bea114151c4c5504b93a3",
-    "ref_origin": "release/1.9"
+    "ref": "62d63a79793e428596c7c631b12db2d7d68ab170",
+    "ref_origin": "integrate/marathon-1.5"
```

Comment:
Please note that the UI is not compatible with Marathon 1.5 without the changes introduced `integrate/marathon-1.5`, I'm ok with landing those changes in separate PRs if that makes it easier to land this PR.